website/docs/docs/on-premises/releases/1.29.0.md

```diff
@@ -0,0 +1,25 @@
+---
+title: 1.29.0
+---
+
+## Release date
+
+31 January 2024
+
+## Features
+
+* Add verification modes to the GET /integration endpoint: replay or comparison along with other integration details
```

Comment:
I don't know what "along with other integration details" means. That can be deleted.